src/fiatExchanges/SelectProvider.tsx

```diff
@@ -187,6 +190,7 @@ export default function SelectProviderScreen({ route, navigation }: Props) {
     asyncExchanges.loading ||
     selectFiatConnectQuoteLoading
 
+  Logger.info(TAG, asyncProviders.result?.externalProviders)
```

Comment:
leftover from debugging? 

Could keep it, but should probably have a little extra context in the log.